src/convertToObject.js

```diff
@@ -3,16 +3,41 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectForStyles = {};
+  const array = [];
+  const arrSplitFirstLevel = sourceString.split(';');
+
+  for (const words of arrSplitFirstLevel) {
+    const arrSplitSecondLevel = words.split(':');
+
+    if (arrSplitSecondLevel[0] !== undefined
+      && arrSplitSecondLevel[1] !== undefined) {
+      const propertyName = arrSplitSecondLevel[0].trim();
+      const propertyValue = arrSplitSecondLevel[1].trim();
+
+      array.push(propertyName);
+      array.push(propertyValue);
```

Comment:
method push can use multiple params 
```suggestion
      array.push(propertyName, propertyValue);
```